### DISCUSSION TITLE
MicrosoftWindowsSdkWin32MetadataBasePath project property works for the Windows.Wdk.winmd file ?

### DISCUSSION THREAD

**Discussion by denis-komarov** [microsoft/CsWin32#1065] [Q&A] [answered]
The [README](https://www.nuget.org/packages/Microsoft.Windows.CsWin32/0.1.635-beta#readme-tab) says that the `MicrosoftWindowsSdkWin32MetadataBasePath` project property can contain the path to the directory with the `Windows.Win32.winmd` file. 
Does this property also work for the file `Windows.Wdk.winmd` ? Or is this property now completely obsolete ?

**Comment by AArnott** [comment, marked as answer]
That property is obsolete, as described [in our release notes](https://github.com/microsoft/win32metadata/releases/tag/v10.2.185-preview).

I can't find a reference to this property in our README file. Can you double-check, and maybe include the surrounding snippet so I can find it?

**Reply by denis-komarov** [reply]
Thanks for the detailed response. This property is mentioned on nuget.org at this [link](https://www.nuget.org/packages/Microsoft.Windows.CsWin32/0.1.635-beta#readme-tab), but it is a very old release.
